### PR TITLE
feat(TransactionsDownloadCSV): add tags column to csv export

### PR DESCRIPTION
**Pull request by snitin315**
Resolve https://github.com/opencollective/opencollective/issues/3814

# Description

Add a `Tags` column on the transactions CSV export, which is blank if it's a contribution or an expense with no tags, and includes the tags used if it's a tagged expense.

# Screenshots

Screenshot at 2021-01-20 16-08-44

### CODE REVIEW

File: components/transactions/TransactionsDownloadCSV.js [hunk 1]
```diff
@@ -44,11 +44,14 @@ const transformResultInCSV = json => {
     `Net Amount (${collectiveCurrency})`,
     'Subscription Interval',
     'Order Date',
+    'Tags',
   ].join(',');
 
   const lines = json.map(i => {
     const profile = `http://opencollective.com/${i.fromCollective.slug}`;
     const subscriptionInterval = i.subscription ? i.subscription.interval : 'one time';
+    const expenseTags = (i.expense && i.expense.category) || '';
```
Comment (snitin315) [inline components/transactions/TransactionsDownloadCSV.js:53]: There is one small issue if we have an expense with multiple tags, only the first tag is returned in `expense.category`.

**Comment by snitin315** [comment]
Friendly ping, /cc @znarf

File: components/transactions/TransactionsDownloadCSV.js [hunk 2]
```diff
@@ -44,11 +44,14 @@ const transformResultInCSV = json => {
     `Net Amount (${collectiveCurrency})`,
     'Subscription Interval',
     'Order Date',
+    'Tags',
   ].join(',');
 
   const lines = json.map(i => {
     const profile = `http://opencollective.com/${i.fromCollective.slug}`;
     const subscriptionInterval = i.subscription ? i.subscription.interval : 'one time';
+    const expenseTags = i.expense && i.expense.tags ? i.expense.tags.join(', ') : '';
```
Comment (SudharakaP) [inline components/transactions/TransactionsDownloadCSV.js:53]: Would do a minor simplification, 

```suggestion
    const expenseTags = i.expense?.tags ? i.expense.tags.join(', ') : '';
```

**Review by SudharakaP — APPROVED** [review]

**Review by znarf — APPROVED** [review]
Looks good.

@SudharakaP can you take responsibility of shipping this?

**Comment by SudharakaP** [comment]
@snitin315 : Works nicely. Thanks much for the contribution. 😄 💯

**Comment by snitin315** [comment]
> @snitin315 : Works nicely. Thanks much for the contribution. 😄 💯

You're welcome 😄

**Pull request merged**